wrapper/src/main/java/software/amazon/jdbc/PluginService.java

```diff
@@ -146,6 +146,16 @@ HostSpec getHostSpecByStrategy(List<HostSpec> hosts, HostRole role, String strat
 
   void forceRefreshHostList(Connection connection) throws SQLException;
 
+  /**
+   * Initiates a topology update.
+   *
+   * @param writerImportant true, if a caller expect to get a topology with the latest confirmed writer
+   * @param timeoutMs timeout in msec to wait till topology get refreshed and a new (or existing) writer is confirmed
+   * @return true, if successful. False, if operation is unsuccessful or timeout is reached
+   * @throws SQLException if there was an error establishing a connection or fetching a topology
+   */
+  boolean forceRefreshHostList(final boolean writerImportant, final long timeoutMs) throws SQLException;
```

Comment:
Can we rename `writerImportant` to `shouldVerifyWriter`? I think its a bit clearer. Same for the other occurrences of `writerImportant` in the PR